.devcontainer/.env

```diff
@@ -1,7 +1,17 @@
+# common service settings.
+FML_DEV_MODE=true
+FML_LOG_LEVEL=DEBUG
+
+# database settings.
+LC_COLLATE=POSIX
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 POSTGRES_DB=postgres
 POSTGRES_HOSTNAME=localhost
-LC_COLLATE=POSIX
-FML_LOG_LEVEL=DEBUG
-FML_DEV_MODE=true
\ No newline at end of file
+
+# aws client settings.
+AWS_ACCESS_KEY_ID: "user"
+AWS_SECRET_ACCESS_KEY: "password"
+
+# s3 compatible storage settings.
+FML_S3_ENDPOINT_URI=http://localhost:9000
```

Comment:
let's group all of these together as `s3 compatible storage settings.`

Reply:
fixed.